ethereum/eth/build.gradle

```diff
@@ -43,7 +43,10 @@ dependencies {
   implementation 'com.google.guava:guava'
   implementation 'io.vertx:vertx-core'
   implementation 'org.apache.tuweni:tuweni-bytes'
+  implementation 'org.apache.tuweni:tuweni-ethstats'
   implementation 'org.apache.tuweni:tuweni-units'
+  implementation 'org.logl:logl-logl'
```

Comment:
Currently our logging library is Log4j2 and only Log4j2.  Can this PR be done without the Logl dependency?

Reply:
Logl proxies to log4j2.